webapp/src/app.tsx

```diff
@@ -4466,6 +4454,13 @@ document.addEventListener("DOMContentLoaded", () => {
         return;
     }
 
+    const query = core.parseQueryString(window.location.href);
+
+    // Handle auth callback redirect.
+    if (query["authcallback"]) {
+        auth.loginCallback(query);
```

Comment:
What's the difference between here and line 4295?

Reply:
Line 4295 was handling the callback hash `#authcallback`, but with the hash, all url params were getting stripped off the url. I couldn't figure out where. I was driving me crazy so I switched to using a url parameter to indicate its an auth callback.